core/modules/node/node.pages.inc

```diff
@@ -394,6 +433,15 @@ function node_form_submit($form, &$form_state) {
     watchdog('content', '@type: updated %title.', $watchdog_args, WATCHDOG_NOTICE, $node_link);
     backdrop_set_message(t('@type %title has been updated.', $t_args));
   }
+  if ($node->published_on) {
+    $message = '';
+    if (!empty($original_status)) {
+      $message .= t('@type has been unpublished.', array('@type' => $type)) . ' ';
+    }
+    $today = date("D M j G:i:s", strtotime($node->future_publish));
```

Comment:
<img width="381" alt="screen shot 2018-04-21 at 3 32 24 pm" src="https://user-images.githubusercontent.com/397895/39089330-54ab0b92-4579-11e8-88a5-ac41de29c625.png">
This date/time format is hard for me to understand. We should use one of the predefined formats so that if a site admin changes from am/pm to 24-hour site-wide (or vice versa) this time will update too. How about we use `short` like so:
```php
$today = format_date($node->future_publish, 'short');
```

Reply:
True.